dotcom-rendering/src/layouts/ShowcaseLayout.tsx

```diff
@@ -551,6 +559,15 @@ export const ShowcaseLayout = (props: WebProps | AppsProps) => {
 								standfirst={article.standfirst}
 							/>
 						</GridItem>
+						<GridItem area="disclaimer">
+							{hasAffiliateLinksDisclaimer && (
+								<Disclaimer
+									html={
+										article.affiliateLinksDisclaimer ?? ''
```

Comment:
Feel free to ignore this one, but I think if you inlined the definition of `hasAffiliateLinksDisclaimer` you would't have to include the `?? ''`